sender6.h

```diff
@@ -23,4 +23,8 @@ struct packet * sender6_send_icmp_fixrev(struct sender6 *sender,
 		size_t padding);
 */
 
+struct packet * sender6_send_tcp(struct sender6 *s, struct libnet_in6_addr dst,
```

Comment:
like the `send_icmp` function, this function should receive values for the TCP packer directly. in particular, it should not receive `flowid`, but should receive the sequence number. this is so the function is generic and can be used in other modules (you should generate the value for the sequence number (encoding flow id, ttl, and sequence) outside this module).